_posts/en/newsletters/2019-03-26-newsletter.md

```diff
@@ -0,0 +1,256 @@
+---
+title: 'Bitcoin Optech Newsletter #39'
+permalink: /en/newsletters/2019/03/26/
+name: 2019-03-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a proposal to encrypt P2P communication
+and describes Lightning Loop, a tool and service for withdrawing
+bitcoins from an LN channel to an onchain transaction.  Also included
+are links to resources about bech32 adoption, summaries of popular
+questions and answers from Bitcoin StackExchange, and a list of notable
+code changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC2:** The second Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  Testing is still needed by organizations and experienced users who
+  plan to run the new version of Bitcoin Core in production.  Use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+## News
+
+- **Version 2 P2P transport proposal:** Jonas Schnelli sent a proposed
```

Comment:
D'oh, can't believe I forgot to put a link.  I'm going to link to the mailing list thread because I want to encourage people to comment there for now.  If we link to it in the future, we may use the gist (but hopefully he'll have a new BIP number by then).